bflibrary/src/x86-win-sdl/sscreen.c

```diff
@@ -626,42 +624,68 @@ TbBool LbHwCheckIsModeAvailable(TbScreenMode mode)
 /** @internal
  * Provides simplified SDL_BlitScaled() functionality for SDL1.
  */
-int LbI_SDL_BlitScaled(SDL_Surface *src, SDL_Surface *dst, int resolutionMul)
+int LbI_SDL_BlitScaled(SDL_Surface *src, SDL_Surface *dst)
 {
-    long i, j;
-    long mdWidth, mdHeight;
 
-    LbScreenLock();
-    LbIPhysicalScreenLock();
+  /* shortcircuit for 1:1 */
```

Comment:
indent too small

Reply:
i used that style a bit - with the leading `/*` 2 characters before the next line of code ... I'll assume you want me to shift those over too unless you say otherwise 👍🏻

Reply:
The code uses both 2 and 4 spaces.. in all recent functions, I use 4.
So, if modifying a function, best to adhere to its current convention. If creating new one, go with 4.